src/RPC/errors.ts

```diff
@@ -51,7 +105,8 @@ export {
   ErrorRpcParse,
   ErrorRpcHandlerFailed,
   ErrorRpcMessageLength,
-  ErrorRpcRemoteError,
+  ErrorRpcMissingResponse,
+  ErrorPolykeyRemote,
   ErrorRpcNoMessageError,
   ErrorRpcPlaceholderConnectionError,
```

Comment:
What is `ErrorRpcPlaceholderConnectionError`?

Reply:
That was a placeholder error to test some logic. It's a placeholder error for when the underlying streampair errors out due to a connection error or something. In that case we don't want to try and send any error messages back through the stream. I'll have to look it over, remove it and determine if any logic needs to be updated.

Reply:
This should be removed, but I also need to update how we decide to send or not send errors over the RPC. I think the only case so far where we don't send a error message is when we can't due to the connection ending. 

So what I can do here is attempt to send all messages, if sending fails due to a connection failure or whatever then we dispatch the error event contain a compound error of what we tried to send and why it failed.

Reply:
The usual way to do this is:

1. Handling errors goes to the connection.
2. Connection errors cannot go to the connection, instead it is logged in the actual agent itself, it should not crash the agent
3. On the receiving side, if it gets a handling error, raise it the normal way.
4. If however on the receiving side, the connection is broken (due to keepalive or something else), then raise a special set of errors regarding the reason why the connection is broken

Reply:
It's important that handling errors AND connection errors does not shutdown the agent. The agent and associated servers should continue to operate.

Reply:
This has been resolved.